libs/common/src/helpers/validation.test.ts

```diff
@@ -149,4 +149,43 @@ describe('validation helpers', () => {
       });
     });
   });
+
+  describe('isMonetaryAmountValid', () => {
+    describe('when monetary amount is valid', () => {
```

Comment:
Is this describe block necessary? These tests actually look like they would parameterise nicely, could have two, one for the success cases and one for the failures to keep this division.

Reply:
I really like how these look now thanks!